test/v8-updates/test-postmortem-metadata.js

```diff
@@ -48,7 +48,8 @@ const missing = getExpectedSymbols().filter((symbol) => {
   return !symbols.includes(symbol);
 });
 
-assert.strictEqual(missing.length, 0, `Missing constants: ${missing}`);
+assert.strictEqual(missing.length, 0,
+                   `Missing constants: \n${missing.join('\n')}`);
```

Comment:
Optional nit suggestion to get rid of a trailing space and indent the missing constants by 2 spaces in the output for easier reading:

```suggestion
                   `Missing constants:\n  ${missing.join('\n  ')}`);
```